app/views/admin/booths/_form.html.haml

```diff
@@ -0,0 +1,33 @@
+.row
+  .col-md-12
+    .page-header
+      %title Request a Booth
+.row
+  .col-md-8
+    = semantic_form_for(@booth,  url: @booth.new_record? ?  admin_conference_booths_path(@conference.short_title) : admin_conference_booth_path(@conference.short_title, @booth.id), html: { multipart: true }) do |f|
+      = f.input :title, as: :string, required: true
+      = f.input :description, input_html: { rows: 5, data: { provide: 'markdown-editable' } }, required: true,
+        hint: 'This field becomes public upon request acceptance'
+      = f.input :reasoning, input_html: { rows: 5, data: { provide: 'markdown-editable' } }, required: true,
+        label: 'How it fits the conference'
+      = f.input :submitter_relationship, input_html: { rows: 5, data: { provide: 'markdown-editable' } }, required: true,
+        label: 'Your personal relationship with the organization you are applying for',
```

Comment:
This label is a bit big, let's think about how to make it shorter. And the hint can be a bit more explanatory, eg. community manager